lib/command/src/Obelisk/Command.hs

```diff
@@ -185,6 +186,14 @@ data DeployCommand
   | DeployCommand_Update
   deriving Show
 
+--TODO: Result should provide normalised path and also original user input for error reporting.
+thunkDirectoryParser :: Parser FilePath
```

Comment:
Please add a haddock and remove this TODO. make an issue for it after we merge this PR